test/system/src/main/java/io/pravega/test/system/framework/services/docker/ZookeeperDockerService.java

```diff
@@ -28,7 +28,7 @@
 @Slf4j
 public class ZookeeperDockerService extends DockerBasedService {
 
-    private static final String ZK_IMAGE = "jplock/zookeeper:3.5.1-alpha";
+    private static final String ZK_IMAGE = "zookeeper:3.5.4-beta";
```

Comment:
Now that we are using the official zookeeper docker image we can also configure the zk logs to stored similar to other services using `ZOO_LOG4J_PROP="INFO,ROLLINGFILE" ` and mounting /logs folder.
ref: https://hub.docker.com/_/zookeeper